src/build/__tests__/build-test.js

```diff
@@ -149,4 +168,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [customDirective] } };
+    const target = build(schemaSource + querySourceWithDirective, config, undefined, false);
```

Comment:
You don't need to pass `false` as the last argument because the schema won't be inferred. Once you get rid of the `false` you can also get rid of the `undefined`.